www/css/style.css

```diff
@@ -806,9 +806,26 @@ button.button.back-button.buttons.button-clear.header-item {
         top: 310px;
         line-height: 16px;
 }
+
+.start-end-addresses-container {
+  display: inline-block;
+  width: calc(100% - 25px);
```

Comment:
To leave more space for the start and end addresses. 
I guess it could get a bit tight the space for addresses in small screen smartphones. Do you agree? Or you think we should leave more space between the addresses and the arrow?